server/organisation_invite.go

```diff
@@ -0,0 +1,84 @@
+package server
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/frain-dev/convoy/server/models"
+	"github.com/frain-dev/convoy/util"
+	"github.com/go-chi/render"
+	log "github.com/sirupsen/logrus"
+)
+
+// InviteUserToOrganisation
+// @Summary Get an organisation
+// @Description This endpoint invites a user to join an organisation
+// @Tags Organisation
+// @Accept  json
+// @Produce  json
+// @Param orgID path string true "organisation id"
+// @Param invite body models.OrganisationInvite true "Organisation Invite Details"
+// @Success 200 {object} serverResponse{data=Stub}
+// @Failure 400,401,500 {object} serverResponse{data=Stub}
+// @Security ApiKeyAuth
+// @Router /organisations/{orgID}/invite_user [post]
+func (a *applicationHandler) InviteUserToOrganisation(w http.ResponseWriter, r *http.Request) {
+	var newIV models.OrganisationInvite
+	err := util.ReadJSON(r, &newIV)
+	if err != nil {
+		_ = render.Render(w, r, newErrorResponse(err.Error(), http.StatusBadRequest))
+		return
+	}
+
+	org := getOrganisationFromContext(r.Context())
+	_, err = a.organisationInviteService.CreateOrganisationMemberInvite(r.Context(), org, &newIV)
+	if err != nil {
+		log.WithError(err).Error("failed to create organisation member invite")
+		_ = render.Render(w, r, newServiceErrResponse(err))
+		return
+	}
+
+	_ = render.Render(w, r, newServerResponse("invite created successfully", nil, http.StatusCreated))
+}
+
+// ProcessOrganisationMemberInvite
+// @Summary Get organisations
+// @Description This endpoint process a user's response to an organisation invite
+// @Tags Organisation
+// @Accept  json
+// @Produce  json
+// @Param token query string true "invite token"
+// @Param email query string true "email"
+// @Param accepted query string true "email"
+// @Param user body models.User false "User Details"
+// @Success 200 {object} serverResponse{data=Stub}
+// @Failure 400,401,500 {object} serverResponse{data=Stub}
+// @Security ApiKeyAuth
+// @Router /process_organisation_member_invite [post]
+func (a *applicationHandler) ProcessOrganisationMemberInvite(w http.ResponseWriter, r *http.Request) {
+	token := r.URL.Query().Get("token")
+	email := r.URL.Query().Get("email")
```

Comment:
Why do we need the `email` here? I'm guessing the `token` on its own should be sufficient.

Reply:
You're right. I have removed it.